console/src/cluster/notification.tsx

```diff
@@ -0,0 +1,37 @@
+// Copyright 2024 Synnax Labs, Inc.
+//
+// Use of this software is governed by the Business Source License included in the file
+// licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with the Business Source
+// License, use of this software will be governed by the Apache License, Version 2.0,
+// included in the file licenses/APL.txt.
+
+import { Button } from "@synnaxlabs/pluto";
+
+import {
+  NotificationAdapter,
+  SugaredNotification,
+} from "@/notifications/Notifications";
+import { OpenUpdateDialogAction } from "@/version/Updater";
+
+export const versionOutdatedAdapter: NotificationAdapter = (status) => {
+  if (status.data == null) return null;
+  if (status.data.type !== "serverVersionMismatch") return null;
```

Comment:
Could we change the string `"serverVersionMismatch"` to a `const` that gets exported from Pluto and imported here?